app/presenters/email_archive_presenter.rb

```diff
@@ -14,6 +14,7 @@ def for_s3(record, archived_at)
       created_at_utc: record.fetch("created_at").utc.strftime(S3_DATETIME_FORMAT),
       finished_sending_at_utc: record.fetch("finished_sending_at").utc.strftime(S3_DATETIME_FORMAT),
       id: record.fetch("id"),
+      marked_as_spam: true,
```

Comment:
Should this be hardcoded as `true`?

Reply:
Oops... it should not be!

Reply:
Well spotted - now amended to the more sensible `record.fetch("marked_as_spam")`.

I've also changed the presenter test to expect false as a value for `marked_as_spam` as I assume we are testing the default non-spam case here.